packages/contracts/plugin-settings.ts

```diff
@@ -2,13 +2,13 @@ import buildMetadata from './src/build-metadata.json';
 import releaseMetadata from './src/release-metadata.json';
 import {VersionTag} from '@aragon/osx-commons-sdk';
 
-export const PLUGIN_CONTRACT_NAME = 'MyPlugin'; // This must match the filename `packages/contracts/src/MyPlugin.sol` and the contract name `MyPlugin` within.
-export const PLUGIN_SETUP_CONTRACT_NAME = 'MyPluginSetup'; // This must match the filename `packages/contracts/src/MyPluginSetup.sol` and the contract name `MyPluginSetup` within.
-export const PLUGIN_REPO_ENS_SUBDOMAIN_NAME = 'my'; // This will result in the ENS domain name 'my.plugin.dao.eth'
+export const PLUGIN_CONTRACT_NAME = 'Admin';
+export const PLUGIN_SETUP_CONTRACT_NAME = 'AdminSetup';
+export const PLUGIN_REPO_ENS_SUBDOMAIN_NAME = 'admin'; // This will result in the ENS domain name 'my.plugin.dao.eth'
```

Comment:
comment should be `admin.dao.eth`

Reply:
Fixed in 803b66d94cd419ec64cc077d8dcc4d9e1b294217